src/pudl/extract/ferc1.py

```diff
@@ -174,6 +174,19 @@
         "dbf": ["f1_income_stmnt", "f1_incm_stmnt_2"],
         "xbrl": "statement_of_income_114",
     },
+    "retained_earnings_ferc1": {
+        "dbf": "f1_retained_erng",
+        "xbrl": "retained_earnings_118",
+    },
+    "retrained_earnings_appropriations_ferc1": {
+        "dbf": "f1_retained_erng",
+        "xbrl": "retained_earnings_appropriations_118",
+    },
```

Comment:
I added this second table in several places... but didn't actually develop the transformation process for it. Perhaps I should remove it from this pr.